docs/source/man.rst

```diff
@@ -513,6 +513,16 @@ For example, the raw facts of a scan that includes both network and vcenter sour
 
   Required. Path to a file location where the report data is saved.
 
+Merging Scan Job Results
+~~~~~~~~~~~~~~~~~~~~~~~~
+The ``qpc report merge`` command combines results from two or more scan jobs.
```

Comment:
`two or more scan jobs to create a report. The ``qpc report summary`` or ``qpc report detail` can be used to obtain the report.

Reply:
@chambridge Fixed